core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/DataRepoExpressionFixture.scala

```diff
@@ -34,22 +34,52 @@ trait DataRepoExpressionFixture {
   val validEntityInputExpressions: Seq[String] = Seq(
     "this.gvcf",
     "this.with-dash",
-    "this.library:cohort",
-    "this.library:cohort1",
-    "this.arbitrary:whatever",
     "this.underscores_are_ok",
     "this._",
     """["foo","bar", this.gvcf]""",
     """["a",{"more":{"elaborate":this.gvcf}}]""",
     """{"more":{"elaborate":{"reference1": this.gvcf, "path":"gs://abc/123"}}}""",
-    """["foo", "bar", 123, ["array", this.gvcf, this.library:cohort], false]""",
-    """["foo", "bar", 123, ["array", this.gvcf, [this.library:cohort]], false, ["abc", this.with-dash]]"""
```

Comment:
Why were these removed (and the others up on lines 37-39)?  I know that the namespaced attributes are not valid for DataRepo expressions, but these are valid expressions for the LocalEntityProvider.   This means we might need to creates some different lists that build for DataRepo vs. Local expressions.

Reply:
I think they are separated. There's a Fixture for local expressions here: https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/LocalExpressionFixture.scala